xbmc/music/tags/TagLibVFSStream.cpp

```diff
@@ -37,6 +39,17 @@ TagLibVFSStream::TagLibVFSStream(const std::string& strFileName, bool readOnly)
   {
     if (!m_file.Open(strFileName))
       m_bIsOpen = false;
+    else
+    { // Check if this stream is a Shoutcast, and skip scanning tags if it is.
+      // Shoutcast has its own metadata format that TagLib does not read
+      CFileCache* fileCacheItem = dynamic_cast<CFileCache*>(m_file.GetImplementation());
```

Comment:
How can I deactivate cache to test?
And do you have any other suggestions how to burrow reliably into the IFile sequence of items to check for the Shoutcast? I am at the limit of my knowledge on this